src/validation.cpp

```diff
@@ -524,7 +524,7 @@ class MemPoolAccept
                             /* m_bypass_limits */ false,
```

Comment:
In d3d653dce34ad1f3235869b8a54f9572e89e5b68  [policy] package rbf 

Update places where it is claimed that package RBF is disabled?


```diff
diff --git a/src/validation.cpp b/src/validation.cpp
index 8372fcadca1..d18fc1c0f2c 100644
--- a/src/validation.cpp
+++ b/src/validation.cpp
@@ -602,8 +602,8 @@ public:
     /**
      * Submission of a subpackage.
      * If subpackage size == 1, calls AcceptSingleTransaction() with adjusted ATMPArgs to avoid
-     * package policy restrictions like no CPFP carve out (PackageMempoolChecks) and disabled RBF
-     * (m_allow_replacement), and creates a PackageMempoolAcceptResult wrapping the result.
+     * package policy restrictions like no CPFP carve out (PackageMempoolChecks)
+     * and creates a PackageMempoolAcceptResult wrapping the result.
      *
      * If subpackage size > 1, calls AcceptMultipleTransactions() with the provided ATMPArgs.
      *
@@ -950,7 +950,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     ws.m_iters_conflicting = m_pool.GetIterSet(ws.m_conflicts);
 
     // Note that these modifications are only applicable to single transaction scenarios;
-    // carve-outs and package RBF are disabled for multi-transaction evaluations.
+    // carve-outs are disabled for multi-transaction evaluations.
     CTxMemPool::Limits maybe_rbf_limits = m_pool.m_opts.limits;
 
     // Calculate in-mempool ancestors, up to a limit.
```

Reply:
done, and found one more place with out of date comment